ui/plugins/pluginpanel.html

```diff
@@ -39,6 +42,7 @@
           </li>
           <li>
             <a ng-if="!job_log_urls.length"
+               id="logviewer-btn"
```

Comment:
I debated about adding this id to only the first, parsed case. But I thought if someone wants to hook something later to the id, it seems it should already there for all cases. Only one id can exist in the dom at any one time, courtesy of the `ng-if`.

If you'd prefer only 1 though, I can remove the last 3 and just apply it on the parsed case.